static/collection.js

```diff
@@ -0,0 +1,47 @@
+(async () => {
+    function clear_all_games_table(){
+        const table = document.getElementById('table');
+        table.innerHTML = "";
+    }
+
+async function create_all_games_table() {
+    clear_all_games_table();
+    const user = document.getElementById("user").value;
+    fetch('/all_games?user=' + user)
+        .then(async response => {
+            if (response.ok) {
+                const games = await response.json();
+                const table = document.getElementById('table')
```

Comment:
Not sure if this would be better with `document.createElement()` instead but I wasn't able to get it working with that.